packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -785,6 +785,23 @@ export class Location {
             this.range = new Range(rangeOrPosition, rangeOrPosition);
         }
     }
+
+    /*---------------------------------------------------------------------------------------------
+    *  Copyright (c) Microsoft Corporation. All rights reserved.
+    *  Licensed under the MIT License. See License.txt in the project root for license information.
+    *--------------------------------------------------------------------------------------------*/
+    // copied from https://github.com/microsoft/vscode/blob/master/src/vs/workbench/api/common/extHostTypes.ts
```

Comment:
We require a CQ for the copied code.
https://github.com/theia-ide/theia/wiki/Registering-CQs

Reply:
Do you have to be an official commiter in order to register a CQ? Following the document linked, I login and go to the project page but there isn't anywhere that says intellectual property or create a contribution questionnaire